.travis.yml

```diff
@@ -9,7 +9,7 @@ before_install:
   - bundle --version
 bundler_args: --without=development
 
-rvm: 2.0.0
+rvm: 2.2.2
```

Comment:
I hope we can pin down some transitive dependencies. I'll fix the build when I find the time.

Thanks anyway for the PR!